spec/posix.test.sh

```diff
@@ -142,3 +142,20 @@ three
 EOF2
 ## stdout-json: "one\ntwo\nthree\n"
 
+#### source works for files in subdirectory
+mkdir -p dir
+echo "echo path" > dir/cmd
+. dir/cmd
+rm dir/cmd
+## STDOUT:
+path
```

Comment:
It looks like this is missing an `# END`.  Although now that I check the parser in `test/sh_spec.py`, END is optional!  That is weird.